packages/params/test/unit/applicationDomains.test.ts

```diff
@@ -8,7 +8,7 @@ describe("validate application domains", () => {
       for (let i = 0; i < DOMAIN_APPLICATION_MASK.length; i++) {
         r += DOMAIN_APPLICATION_MASK[i] & domain[i];
       }
-      expect(r).to.be.above(0, `${name} mask application should be valid`);
+      expect(r).toBeGreaterThan(0);
```

Comment:
could keep comment, from the assertion it is not instantly obvious that this means the mask is valid